lib/core/rancher/step.sh

```diff
@@ -32,7 +32,22 @@ function ih::setup::core.rancher::install() {
 
     cp "${THIS_DIR}/io.rancherdesktop.profile.defaults.plist" "$HOME/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
 
-    if command -v rdctl; then
+    # If rancher is already installed  reset to factory
+    if [ -d "/opt/homebrew/Caskroom/rancher" ] || [ -d "/opt/homebrew/Caskroom/ih-rancher" ] 
+    then
+        $(/Applications/Rancher\ Desktop.app/Contents/Resources/resources/darwin/bin/rdctl factory-reset)
+    fi
+    
+    # Check if Rancher was installed manually with brew
+    if [ -d "/opt/homebrew/Caskroom/rancher" ] 
+    then
```

Comment:
Same thing here. But really we should factory-reset before uninstalling either of them, I'm not sure that an uninstall cleans up everything.

Reply:
I uninstall both packages and the profile file remains there.